oscillogram_classification/cluster.py

```diff
@@ -173,7 +175,35 @@ def create_dataset(norm: bool, data_path: str) -> None:
     create_processed_time_series_dataset(data_path, norm)
 
 
-def create_processed_time_series_dataset(data_path: str, norm: bool) -> None:
+def clean_incorrect_patches(paths: list) -> list:
+    """
+    Removes time series that were not split into the expected number of patches (either too many or too little).
+
+    5 patches are expected for positive measurements and 3 for negative ones.
+
+    :param paths: list of paths to all patches. If negative patches are included, their path should contain the word "negative".
```

Comment:
consistency: no ".", I would suggest: "list of paths to all patches; if negative patches are included, their path should contain the word "negative"

Reply:
Also, the line exceeds 120 chars